packages/store-sync/src/createStoreSync.ts

```diff
@@ -61,7 +64,7 @@ export async function createStoreSync({
   maxBlockRange,
   initialState,
   initialBlockLogs,
-  indexerUrl,
+  indexerUrl: indexerUrlInput,
```

Comment:
small nit: I tend to name these "initial"
```suggestion
  indexerUrl: initialIndexerUrl,
```